modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
+channel. For patch and CVE fixes this can range from several hours to a day and
+allows an extra period of assessment in how the software performs. If issues are
+detected during rollout, upgrades to that version may be blocked in both the fast
+and stable channels, and a new version may be introduced that will be the new
+preferred upgrade target.
+
+Customers can improve this process by configuring pre-production systems on the
+fast channel, production systems on the stable channel, and participating in Red
+Hat’s connected customer program - this allows Red Hat to observe the impact of
+updates on your specific hardware and software configurations. Future releases may
+improve or alter the pace at which updates move from the fast to the stable channels.
+
+If issues are discovered with an upgrade between patch levels, Red Hat may withdraw
+that suggested upgrade for affected versions. A newer patch would become available
+in the appropriate channels and be suggested for upgrade.
+
+= Upgrade Version Paths
+
+{product-title} maintains an upgrade recommendation service that understands the
+version of {product-title} you have installed as well as the path to take within
+the channel you choose to get you to the next release. You can imagine seeing the
+following in the fast-{product-version} channel:
+
+* {product-version}.0
+* {product-version}.1
+* {product-version}.3
+* {product-version}.4
+
+The service only recommends upgrades that have been tested and have no known issues.
+If you are on {product-version}.1 and {product-title} is allowing you to select
+{product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise,
+the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no
+longer suggests upgrading to a known vulnerable version. If an issue is found that
+results in a new version being retracted from the recommendations, Red Hat will
+release a new version that is capable of upgrading from all necessary versions,
+including the retracted version.
+
+= Disconnected Clusters
+
+Customers which have chosen to not be connected to Red Hat and are curating their
+own {product-title} container image content manually should consult the Red Hat
+errata associated with product releases and note any comments impacting upgrades.
+During upgrade the user interface may caution about switching between these versions
+and it is up to the customer to ensure they have correctly selected the appropriate
+version before bypassing those cautions.
+
+= Switching Between Channels
+
+It is supported for customers to switch between the fast and stable channel at any
+time. Channels only offer suggested upgrades, and will never suggest a dangerous
+upgrade. If you switch to the candidate channel after installing from a GA version,
+you will see a warning the current version is not recognized, and you can safely
```

Comment:
This is not true, all of our GA releases are also in the candidate channel.  But it matches the existing KB article, so I would rather not evolve this language in this PR (we can improve the wording in follow-up work).

[1]: https://access.redhat.com/articles/4495171